test/transcoding/SPV_INTEL_variable_length_array/complex-cfg.ll

```diff
@@ -21,6 +21,25 @@ newFuncRoot:
   %temp = alloca i32, align 4
   br label %fallthru
 
+; CHECK-LABEL: fallthru
+; CHECK-LLVM: %"ascastB$val41" = alloca i32, i64 %div.3
+fallthru:                          ; preds = %newFuncRoot
```

Comment:
This one's a simpler case: this block which dominates most of the function was originally near the end, so there isn't any way to get around moving it. The test is written in a style where CHECK lines are next to the corresponding input code. Since I don't think this test is intended to test dominator ordering, it seems clearer to just move this block up so that it doesn't need to be reordered rather than moving just the CHECK lines by themselves.